src/data/wallets/wallet-filters.ts

```diff
@@ -131,8 +131,9 @@ const walletFilterData: { [key: string]: WalletFilter } = {
     filterKey: "erc_20_support",
   },
   eip_1559_support: {
-    title: "EIP-1559 support",
-    description: "Lower gas fees with type-2 transactions",
+    title: "Fee optimization",
+    description:
+      "Support type 2 transactions for optimized gas fees, and fee refunds for unused gas",
```

Comment:
Questions (that could be solved in a separate PR):
- Would it still be illuminating/valuable to mention EIP 1559 anywhere?
- Do we define "type 2 transactions" anywhere? If not we probably should... e.g. on the tx page: https://ethereum.org/en/developers/docs/transactions/#typed-transaction-envelope

Reply:
Created an issue around the 2nd point: https://github.com/ethereum/ethereum-org-website/issues/7372

Reply:
I think we could add something to the description for this. I was going to do a general pass over the copy for these next sprint so I think it could be done then.